src/cluster/pager.rs

```diff
@@ -51,14 +52,15 @@ impl<
             pager: self,
             pager_state: state,
             query,
+            qv
         }
     }
 
-    pub fn query<Q>(&'a mut self, query: Q) -> QueryPager<'a, Q, SessionPager<'a, M, S, T>>
+    pub fn query<Q>(&'a mut self, query: Q, qv: Option<QueryValues>) -> QueryPager<'a, Q, SessionPager<'a, M, S, T>>
```

Comment:
It will be a breaking change in the API which will require a new major version release. Can you please add a new method (`query_with_values` or any other descriptive name) with a proposed signature and keep `query` in a previous shape?

Reply:
I read in this comment you are thinking about a new major release: https://github.com/AlexPikalov/cdrs/pull/344#issuecomment-652073197. Is it still needed to remain backwards compatible? I think the new parameters should always be explicitly filled.

Reply:
Note: I made a new commit which adds a consistency level parameter and updated the 'Any' consistency documentation, since the documentation was wrong